Kudu.FunctionalTests/Jobs/WebJobsTests.cs

```diff
@@ -285,6 +288,95 @@ public void TriggeredJobTriggers()
         }
 
         [Fact]
+        public void ScheduledTriggeredJobTriggers()
+        {
+            RunScenario("ScheduledTriggeredJobTriggers", appManager =>
+            {
+                const string jobName = "job1";
+
+                TestTracer.Trace("Copying the script to the triggered job directory");
+
+                appManager.JobsManager.CreateTriggeredJobAsync(jobName, "run.cmd", JobScript).Wait();
+
+                // Wait for file watcher to pick up
+                Thread.Sleep(30 * 1000);
```

Comment:
I wasn't suggesting not to have a test of course. I was asking whether you could use a better wait strategy. Initially I thought this test was running against local file system, which would mean file events would be very fast.